packages/boinc/no-alt-platforms.patch

```diff
@@ -0,0 +1,18 @@
+diff -uNr boinc-client_release-7.16-7.16.11/m4/boinc_platform.m4 boinc-client_release-7.16-7.16.11.mod/m4/boinc_platform.m4
+--- boinc-client_release-7.16-7.16.11/m4/boinc_platform.m4	2020-08-30 16:04:59.000000000 +0800
++++ boinc-client_release-7.16-7.16.11.mod/m4/boinc_platform.m4	2020-09-11 11:42:44.110000000 +0800
+@@ -53,14 +53,12 @@
+         ;;
+       aarch64*linux-android*)
+         boinc_platform="aarch64-android-linux-gnu"
+-        boinc_alt_platform="arm-android-linux-gnu"
```

Comment:
This signifies that 32-bit code can also be run on 64-bit platforms, not working?

Reply:
I am a bit worried about the transition to the new package format which requires full proot usage and that currently breaks 32bit code on 64bit systems as shown in the proot issue above.

Currently boinc version 7.16.7, 32bit code is not working out of the box anyway on 64bit as
1. user need to `unset LD_PRELOAD` (I have a patch that automatically does this)
2. user need to add `<alt_platform>arm-android-linux-gnu</alt_platform>` to `cc_config.xml`

Removing this patch solves no2 but still have to deal with no1 (and proot compatibility in new package format)

I am actually the author of this code but I am conflicted with the upstream decision to add it in anyway. Its possible 32bit issue can be fix from the proot side when the new format happens but no guarantees as to when. I think its better for user to opt in (using the 2 steps) rather than opt out of running 32bit code.